packages/idyll-cli/bin/cmds/clean.js

```diff
@@ -0,0 +1,46 @@
+const fs = require('fs-extra');
+const inquirer = require('inquirer');
+exports.command = 'clean';
+exports.description = 'Remove .idyll folder';
+
+const PATH = './.idyll';
+
+exports.handler = async _ => {
+  try {
+    let isTokenExists = await checkIfTokenExists();
+    if (isTokenExists) {
+      let userAnswer = await confirm();
+      if (userAnswer === 'Yes') {
+        removeIdyll();
+      }
+    } else {
+      removeIdyll();
+    }
+  } catch (e) {
+    console.log(e);
+  }
+};
+
+checkIfTokenExists = _ => {
```

Comment:
I think this should be `const checkIfTokenExists = ...`

Reply:
done !!